css-three-column-layout/styles.css

```diff
@@ -0,0 +1,53 @@
+body {
+  color: white;
+  background-color: #1996da;
+  font-family: Lato, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  max-width: 800px;
+  margin: auto;
+}
+
+#motto {
+  font-size: 1.5rem;
+  margin-left: 1rem;
+}
+
+.imagediv {
+  background-color: white;
+  border-radius: 50%;
+  width: 6rem;
+  display: flex;
+  margin-bottom: 1rem;
+}
+
+.container,
+.column,
+.row {
+  display: flex;
+  align-items: center;
+}
+
+.column,
+.container {
+  flex-direction: column;
+}
+
+img {
+  width: 100%;
+  scale: 50%;
```

Comment:
Does this actually do anything? I've never seen `scale` used like this. It's usually coupled with a `transform: scale()`

Reply:
<img width="1440" alt="Screenshot 2023-03-30 at 4 31 35 PM" src="https://user-images.githubusercontent.com/20610202/228987250-5f12984b-5a17-480a-bc5a-1957d12e79ab.png">

Without scale, it worked so I just went with it.